client/web/compose/src/components/PageBlocks/ChartConfigurator.vue

```diff
@@ -117,16 +117,12 @@ export default {
       return this.charts.find(({ chartID }) => chartID === this.options.chartID)
     },
 
-    linkTo () {
-      return !this.charts.length || !this.selectedChart ? 'admin.charts' : 'admin.charts.edit'
+    chartExternalLink () {
+      return !this.selectedChart ? 'admin.charts' : 'admin.charts.edit'
```

Comment:
One minor thing (personal preference for readability). Try to use the same condition as you do below
So instead of one having `!this.selectedChart` and the other just `this.selectedChart`. Try to keep both the same